### PR TITLE
chore(docs): document new snappy vrl functions

### PR DESCRIPTION
**Pull request by MichaHoffmann**
Added documentation for new snappy VRL functions.

### CODE REVIEW

**Review by urseberry — APPROVED** [review]

**Comment by pront** [comment]
Waiting on https://github.com/vectordotdev/vrl/pull/543

File: website/cue/reference/remap/functions/decode_snappy.cue [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package metadata
```
Comment (pront) [inline website/cue/reference/remap/functions/decode_snappy.cue:1]: There are some minor whitespaces issues:

- ` find ./website -name '*.cue' -exec cue fmt {} \;`
- ` make check-docs` to confirm

Comment (MichaHoffmann) [reply website/cue/reference/remap/functions/decode_snappy.cue:1]: done!

**Comment by MichaHoffmann** [comment]
@pront Checks seem to fail because the version of VRL was not bumped in this PR i think, right? Should I bump it to a nightly version?

**Comment by pront** [comment]
> @pront Checks seem to fail because the version of VRL was not bumped in this PR i think, right? Should I bump it to a nightly version?

Ah yes, this is a common problem. You can comment out the examples. I will uncomment them before the next release.

**Comment by MichaHoffmann** [comment]
@pront my bad; i commented too much out and it complained that examples were missing

**Review by pront — APPROVED** [review]

**Comment by MichaHoffmann** [comment]
I wasnt sure how to fix the errors here; empty examples error out and no examples error out too. non-empty examples error because the vrl version is not included. I think there is no way to get this green without bumping vrl, right?

**Comment by pront** [comment]
> I wasnt sure how to fix the errors here; empty examples error out and no examples error out too. non-empty examples error because the vrl version is not included. I think there is no way to get this green without bumping vrl, right?

Yes, this is correct. Let's make this a draft and re-open it after we bump the VRL crate version. Thank you again for contributing this!

**Comment by pront** [comment]
Note we have plans to greatly simplify this process and enhance developer experience.

**Comment by MichaHoffmann** [comment]
> Note we have plans to greatly simplify this process and enhance developer experience.

Oh no worries, im just happy you took the snappy stuff in !

**Pull request merged**